stats/templates/application_stats.html

```diff
@@ -189,6 +201,42 @@ <h3>Confirmed only</h3>
                     }
                 }
             });
+            c3.generate({
+                bindto: '#origin_stats',
+                data: {
+                    json: data['origin'],
+                    keys: {
+                        x: 'origin',
+                        value: ['applications']
+                    },
+                    type: 'bar'
+
+                },
+
+                axis: {
+                    x: {
+                        type: 'category'
+                    }
+                }
+            });
+            c3.generate({
+                bindto: '#origin_stats_confirmed',
+                data: {
+                    json: data['origin_confirmed'],
+                    keys: {
+                        x: 'origin',
+                        value: ['applications']
+                    },
+                    type: 'bar'
+
+                },
+
+                axis: {
+                    x: {
+                        type: 'category'
+                    }
+                }
+            });
```

Comment:
Accepting, but this may get difficult to read when there are a lot of people from diverse origins. 

A suggestion would be to show top 5-10 in the graph + the rest as a table (hidden by default, toggle to see it).